csvcubed/tests/behaviour/inspectcli.feature

```diff
@@ -1,12 +1,19 @@
 Feature: Behaviour testing of csvcubed inspect.
 
-    Scenario: inspect should produce printable for data cube metadata json-ld input
-        Given the existing test-case file "cli/inspect/datacube.csv-metadata.json"
-        When the existing Metadata file exists "cli/inspect/datacube.csv-metadata.json"
+    Scenario: inspect should produce the expected printable for data cube metadata json-ld input of type multi-unit multi-measure.
+        Given the existing test-case file "cli/inspect/multi-unit_multi-measure/alcohol-bulletin.csv-metadata.json"
+        And the existing test-case file "cli/inspect/multi-unit_multi-measure/alcohol-bulletin.csv"
+        And the existing test-case file "cli/inspect/multi-unit_multi-measure/alcohol-content.table.json"
+        And the existing test-case file "cli/inspect/multi-unit_multi-measure/alcohol-sub-type.table.json"
+        And the existing test-case file "cli/inspect/multi-unit_multi-measure/clearance-origin.table.json"
+        
+        When the existing Metadata file exists "cli/inspect/multi-unit_multi-measure/alcohol-bulletin.csv-metadata.json"
```

Comment:
Do we need the following two statements?

Reply:
These are need as whilst existing file exists check (i.e. https://github.com/GSS-Cogs/csvcubed/blob/main/csvcubed-devtools/csvcubeddevtools/behaviour/file.py#L68) checks the presence of the file, it does not set file path to the test context. I will need below two steps for getting the file path and validating its existence:

```@When('the Metadata file path is detected and validated "{csvw_metadata_file}"')
def step_impl(context, csvw_metadata_file: str):
    context.csvw_metadata_json_path = (
        get_context_temp_dir_path(context) / csvw_metadata_file
    )
    path = Path(context.csvw_metadata_json_path)
    assert path.exists() is True

@When('the csv file path is detected and validated "{csv_file}"')
def step_impl(context, csv_file: str):
    context.csv_file_path = (
        get_context_temp_dir_path(context) / csv_file
    )
    path = Path(context.csv_file_path)
    assert path.exists() is True```